_posts/2014-07-29-installing.md

```diff
@@ -263,51 +269,56 @@ Point your browser to http://your-server-hostname/mailscanner/ - you should be p
 - Update the GeoIP database
   Click on the 'Other' menu and select 'Update GeoIP database' and click 'Run Now'.
 
-### Optional steps for Sendmail
+### Optional steps for Exil or Sendmail
 
 #### Setup Sendmail Queue Watcher
 
 You can get MailWatch to watch and display your sendmail queue directories.
 
-Edit `tools/Sendmail_queue/mailq.php` to change the require line to point to the location of `functions.php`; copy `mailq.php` file to `/usr/local/bin` and setup the cronjob:
+copy the `tools/Sendmail-Exim_queue/mailwatch_sendmail_queue.php` file in `/usr/local/bin/`
+
+Verify if the file `mailwatch_sendmail_queue.php` is executable. If not, use "chmod +x" to correct this problem.
+
+setup the cronjob:
 
 ```shell
- $ cp tools/Sendmail_queue/mailq.php /usr/local/bin
```

Comment:
a copy/paste `cp` command is useful when following documentation, readd this line (with correct path)